content/applications/sales/subscriptions/scheduled_actions.rst

```diff
@@ -0,0 +1,174 @@
+
+To set up *Deferred Expense* and *Deferred Revenue* accounts, navigate to :menuselection:`Accounting
+app --> Configuration --> Settings`. Both accounts can be configured in the :guilabel:`Default
+Accounts` section.
+
+.. image:: scheduled_actions/deferred-settings-accounting.png
+   :align: center
+   :alt: The necessary deferred account settings in the Odoo Accounting app's settings page.
+
+Once the necessary information and configurations are entered in the :guilabel:`Deferred Expense`
+and :guilabel:`Deferred Revenue` fields, click :guilabel:`Save` in the upper-left corner.
+
+Create invoice
+--------------
+
+The main function of the :guilabel:`Sale Subscription: generate recurring invoices and payments`
+scheduled action revolves around generating recurring invoices for subscriptions.
+
+Elements related to this scheduled action can be found on confirmed subscription sales orders.
+
+To examine these elements, open any confirmed sales order in the *Subscriptions* application to
+reveal the subscription sales order form.
+
+On a confirmed subscription sales order form, focus on the :guilabel:`Recurring Plan` and
+:guilabel:`Date of Next Invoice` fields.
+
+.. image:: scheduled_actions/confirmed-subscription-sales-order-fields.png
+   :align: center
+   :alt: A confirmed subscription sales order in the Odoo Subscriptions application.
+
+The scheduled action creates an invoice when today's date is the same date as the :guilabel:`Date of
+Next Invoice`.
+
+Odoo uses the information in the :guilabel:`Recurring Plan` field to update the next invoice date
+accordingly.
+
+.. warning::
+   If there is no payment provider set up, Odoo will **not** create an invoice. Instead, in the
+   *chatter* of the subscription sales order, the following message appears: `Automatic renewal
+   succeeded. Free subscription. Next invoice: [date]. No email sent.`
+
+Once the invoice for the subscription sales order is created, the invoice can be viewed by clicking
+the :guilabel:`Invoices` smart button that appears at the top of the subscription sales order.
+
+An email is sent to the customer notifying them of the recurring subscription charge, *if* there is
+a :guilabel:`Payment Token` on the account.
+
+To check if there is a :guilabel:`Payment Token`, open the :guilabel:`Other Info` tab, and look at
+the :guilabel:`Payment Token` field, under the :guilabel:`Subscription` section.
+
+.. image:: scheduled_actions/payment-token-field.png
+   :align: center
+   :alt: The Payment Token field under the Other Info tab on a subscription sales order form.
+
+If there is no :guilabel:`Payment Token` (i.e. payment provider is `Bank Transfer`), the invoice is
+created and sent to the customer. The payment **must** be registered manually in this case.
+
+Closing invoices
+----------------
+
+The :guilabel:`Sale Subscription: generate recurring invoices and payments` scheduled action also
+has the ability to close a subscription, if the following occurs:
```

Comment:
nitpick since this is an array of conditional logic 😛 

```suggestion
has the ability to close a subscription, if the following conditions are met:
```